nbextensions/bulkImportCell/tabs/multiAppInfo.js

```diff
@@ -0,0 +1,158 @@
+define([
+    'bluebird',
+    'common/html',
+    'common/ui',
+    './fileTypePanel',
+    'common/cellComponents/tabs/infoTab',
+], (Promise, html, UI, FileTypePanel, InfoTab) => {
+    'use strict';
+    const cssBaseClass = 'kb-bulk-import-info';
+
+    /**
+     * This widget is responsible for providing info for each of the bulk import apps.
+     *
+     * It maintains the state of the currently shown app info (by file type)
+     *
+     * @param {object} options has keys:
+     *     bus: message bus
+     *     model: cell metadata, contains such fun items as the app parameters and state
+     *     typesToFiles: map from object type to appId and list of input files
+     * @returns
+     */
+    function MultiAppInfoWidget(options) {
```

Comment:
much of this module is similar to the `configure` module, but much simpler, since there's no validation of input fields, etc.